pkg/replication/replication.go

```diff
@@ -145,6 +147,9 @@ func (c *Config) AddRule(opts Options) error {
 	if c.Role == "" {
 		c.Role = arnStr
 	}
+	if opts.ReplicateSync == "sync" {
```

Comment:
Let's return an error if replicate sync is not recognized, also let me make a public constant for "sync" and "async" so mc or other can uses it.

Reply:
done